sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/ScannerForMSBuild.java

```diff
@@ -135,13 +139,29 @@ public boolean isDebugLogs() {
   public ScannerForMSBuild setScannerVersion(String s) {
     checkArgument(!isEmpty(s), "version must be set");
     if (s.equals(LATEST_RELEASE)) {
-      s = gitHub.getLatestScannerReleaseVersion();
+      try {
+        s = gitHub.getLatestScannerReleaseVersion();
+      } catch (IllegalStateException e) {
+        s = getLatestScannerVersionFromArtifactory()
+          .orElseThrow(() -> new IllegalStateException("Failed to resolve latest version of Scanner for MSBuild"));
+        LOG.atLevel(Level.DEBUG).log("Failed to retrieve latest version from GitHub, using version from Artifactory", e);
```

Comment:
NItpick: This can be simplified with LOG.atDebug().log

Reply:
done